### ISSUE TITLE
Won't let me import

### ISSUE THREAD

**Issue by fromtheneck** [codelucas/newspaper#182]
```
Python 3.4.3 (v3.4.3:9b73f1c3e601, Feb 23 2015, 02:52:03) 
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import newspaper
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/newspaper/__init__.py", line 10, in <module>
    from .article import Article, ArticleException
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/newspaper/article.py", line 155
    print 'You must download() an article before parsing it!'
                                                            ^
SyntaxError: invalid syntax

```

**Comment by yprez** [comment]
Seems like you're installing the Python2 version on Python3.
Use:
```pip install newspaper3k```

**Comment by yprez** [comment]
@codelucas it seems that a lot of people is having this problem. Maybe put this note somewhere higher in the readme?

**Comment by fromtheneck** [comment]
Actually the reason I installed the wrong version was because the Read the Docs gave instructions to install the Python2 version

**Comment by yprez** [comment]
@fromtheneck I see, it makes sense then... I'll try fixing the docs, they haven't been updated for over a year.

p.s. if you don't mind, please close this issue if you got it to work...

**Comment by yprez** [comment]
I submitted a doc fix in #184

**Comment by codelucas** [comment]
Thanks everyone, the docs are really outdated and bad, they need work :(
